Generator/generator.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+SERVER=localhost
+PORT=2003
+
+dockerize -wait tcp://localhost:${PORT}  -timeout 2m
+for (( i=1; i <= 10; i++ )) do
+	R=$((RANDOM % 100))
+	echo my.awesome.path $R `date +%s` | nc ${SERVER} ${PORT} -w 1
```

Comment:
so if I understand correctly this is the only place where graphite strings are generated currently but this is the only place where they all fall under the same pattern (*)